docs/usage_examples.md

```diff
@@ -0,0 +1,88 @@
+### Usage Examples
+
+Foyer supports atomtyping of both all-atom and coarse-grained molecular systems, and
+also allows for separate force fields to be used to atom-type separate portions of a
+molecular system.
+
+#### Creating a box of ethane
+Here we use mBuild to construct a box filled with ethane molecules and use Foyer to
+atom-type the system, applying the OPLS force field, and save to run-able GROMACS 
+files.
+```python
+import mbuild as mb
+from mbuild.examples.ethane.ethane import Ethane
```

Comment:
I think you can just do `from mbuild.examples import Ethane`. We pulled up a few of the main examples into that namespace.